api/client/build.go

```diff
@@ -315,3 +277,295 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	}
 	return err
 }
+
```

Comment:
All this is a lot of code to be added _just_ for the dockerfile from rewrite, which will be obsolete after client side builder anyways, it just feels kinda gross tbh sorry